aries-site/src/pages/components/card.mdx

```diff
@@ -51,9 +59,64 @@ A card by nature has limited real-estate, so it should present summarized inform
 
 ## When to use
 
+- When presenting high-level, summarized pieces of information.
+- When presenting an item or collection of items that can best be consumed visually.
+
 ## When not to use
 
-- Do not use a Card to filter data on the same screen, instead use **\_**.
+- Do not use a Card as a clickable region to filter data on the same screen.
+- Do not use a Card to group complex content into a single container.
+- When data attributes across items needs to be analyzed, sorted, compared and filtered, a Card may not be the best tool. Instead, [DataTable](/components/datatable) may be more appropriate.
+
+<BestPracticeGroup>
```

Comment:
I thought I sent this on Friday but it looks like my comment was never submitted.. smh

Reply:
Thanks, yes need to look into the search icon.

Re: company logos -- I've swapped Att for the generic 'Acme' chris used in his discover screens. For the others, I think they might be okay because Chris used them in his discover screens.